deploy/charts/cert-manager/templates/webhook-serviceaccount.yaml

```diff
@@ -4,6 +4,10 @@ kind: ServiceAccount
 metadata:
   name: {{ include "webhook.fullname" . }}
   namespace: {{ .Release.Namespace | quote }}
+  annotations:
+  {{- if .Values.serviceAccount.annotations }}
+{{ toYaml .Values.serviceAccount.annotations | indent 4 }}
+  {{- end }}
```

Comment:
I'm not sure if we should set the annotations on the webhook component as well - I think setting it on just the controller would be sufficient, and the webhook's annotations should probably come from `.Values.webhook.serviceAccount.annotations` if we do need to set it 😄